templates/manager/rbac.yaml

```diff
@@ -82,4 +82,12 @@ rules:
   -   apiGroups: [factoryplus.app.amrc.co.uk]
       resources: [kerberos-keys/status]
       verbs: [list, get, watch]
-{{- end -}}
\ No newline at end of file
+{{- end}}
+---
+{{- if .Values.edo.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: edge-deployment-operator
```

Comment:
I don't think this ServiceAccount is added to any RoleBindings. The Manager account had a ClusterRoleBinding, but I'm not sure if that was necessary, we might be able to make do with a RoleBinding.

The account needs the same get/create/patch rights as the manager had.